ui/src/parseError.ts

```diff
@@ -159,29 +158,40 @@ function getCallstack(error: { stack?: string }): string | undefined {
     // tslint:disable-next-line: strict-boolean-expressions
     const stack: string = error.stack || '';
 
-    // Standardize to using '/' for path separator for all platforms
-    let result: string = stack.replace(/\\/g, '/');
-
-    // Standardize newlines
-    result = result.replace(/\r\n/g, '\n');
-
-    // Get rid of the redundant first lines "<errortype>: <errormessage>", start at first line beginning with "at"
-    const atMatch: RegExpMatchArray | null = result.match(/^\s*at\s.+/m);
-    result = atMatch ? result.slice(atMatch.index) : '';
-
-    // Remove the first part of the paths (up to "/{extensions,repos,src/sources,users}/xxx/"), which might container the username.
-    // e.g.:
-    //   (C:\Users\MeMyselfAndI\.vscode\extensions\msazurermtools.azurerm-vscode-tools-0.4.3-alpha\dist\extension.bundle.js:1:313309)
-    //   ->
-    //   (../extensions/msazurermtools.azurerm-vscode-tools-0.4.3-alpha/dist/extension.bundle.js:1:313309)
-    result = result.replace(/([\( ])[^() ]*\/(extensions|[Rr]epos|[Ss]rc|[Ss]ources|[Ss]ource|[Uu]sers|[Hh]ome)\/[^/):\r\n ]+\//g, '$1');
-
-    // Trim each line, including getting rid of 'at'
-    result = result.replace(/^\s*(at\s)?\s*/mg, '');
-    result = result.replace(/\s+$/mg, '');
+    const minifiedLines: (string | undefined)[] = stack
+        .split(/(\r\n|\n)/g) // split by line ending
+        .map(l => {
+            let result: string = '';
+            // Get just the file name, line number and column number
+            const fileMatch: RegExpMatchArray | null = l.match(/[^\/\\\(\s]+\.(t|j)s:[0-9]+:[0-9]+/i);
```

Comment:
It would be very helpful if you put an example line into the comments

Reply:
Isn't that what unit tests are for? 😂

Reply:
No.  :-)